sdk/python/featurestore_sample/setup-resources.sh

```diff
@@ -0,0 +1,36 @@
+pip install --upgrade jupytext
+
+# <create_variables>
+SUBSCRIPTION_ID=$(az account show --query id -o tsv)
+LOCATION=$(az ml workspace show --query location -o tsv)
+RESOURCE_GROUP=$(az group show --query name -o tsv)
+AML_WORKSPACE_NAME=$(az configure -l --query "[?name=='workspace'].value" -o tsv)
```

Comment:
Will workspace be the same for each run?

Reply:
yes, will be the same for a month..

Reply:
I see, feature store today does not allow asset re-registration over the same version. So we would need to find a way to change the asset version each time the notebook is run

Reply:
Discussed offline, will use dynamic versioning when we enable rest of the notebooks